wwwroot/inc/database.php

```diff
@@ -4400,6 +4407,24 @@ function rebuildTagChainForEntity ($realm, $entity_id, $extrachain = array(), $r
 		addTagForEntity ($realm, $entity_id, $tag_id);
 		$result = TRUE;
 	}
+
+	// remove Rack thumnail if Rack or Object tag changes
+	if($result && ( $realm == 'rack' || $realm == 'object'))
+	{
+		$dbxlink->beginTransaction();
```

Comment:
Does a transaction really need to be around this block or those fragments were copied from `commitUpdateTag()`?